### ISSUE TITLE
filterlog: (null) label instead of 0

### ISSUE THREAD

**Issue by xbb** [opnsense/core#5126]
OPNsense version 21.1.9

I'm seeing (null) labels in filterlog, I'm not sure if it's intended.

For example:

```
6,,,(null),igb0_vlan220,match,rdr,in,4,0x0,,64,33783,0,DF,17,udp,60,10.0.220.105,127.0.0.1,60969,53,40
```

It's from a NAT port forward rule without linked rule, this is how it looks like in the UI

![image](https://user-images.githubusercontent.com/239074/127564233-50c108f1-669a-484f-9188-696594ea5ba8.png)

the (null) link points to `/firewall_rule_lookup.php?rid=(null)`

**Comment by fichtner** [comment]
Thanks. NAT rules do not support labels so we need to teach filterlog to return "0" instead of "(null)".

**Comment by fichtner** [comment]
https://github.com/opnsense/ports/commit/5ec9af7edf will fix it in 21.7.1.


Cheers,
Franco
